### ISSUE TITLE
Bug in pagination for first page

### ISSUE THREAD

**Issue by ManiMozaffar** [pydantic/FastUI#222]
check first page html tag,
```html
<a href="?country=CHN&amp;page=2" class="page-link" aria-disabled="false">1</a>
```
so it was rendered wrong, because href should be ?page=1 right?

<img width="1184" alt="Screenshot 2024-02-27 at 13 03 33" src="https://github.com/pydantic/FastUI/assets/86913668/10dbca8a-5552-4b45-b506-f9e1f19d980e">

**Comment by samuelcolvin** [comment]
I've seen a few problems with pagination, definitely something we need to fix

**Comment by samuelcolvin** [comment]
I'll try to fix it soon.

**Comment by hramezani** [comment]
I think it got fixed in https://github.com/pydantic/FastUI/commit/e1203d6b95af18163531ae79b8700fda0c5d37f6

**Comment by ManiMozaffar** [comment]
> I think it got fixed in [e1203d6](https://github.com/pydantic/FastUI/commit/e1203d6b95af18163531ae79b8700fda0c5d37f6)

I'm using CDN, not sure if it's there yet.
https://cdn.jsdelivr.net/npm/@pydantic/fastui-prebuilt@0.0.22/dist/assets/index.js

**Comment by ManiMozaffar** [comment]
Yes tested and it's working fine. thx :)
